hw/mcu/nordic/nrf5340_net/include/mcu/cmsis_nvic.h

```diff
@@ -0,0 +1,28 @@
+/* mbed Microcontroller Library - cmsis_nvic
+ * Copyright (c) 2009-2011 ARM Limited. All rights reserved.
+ *
+ * CMSIS-style functionality to support dynamic vectors
+ */
+
+#ifndef _CMSIS_NVIC_H_
+#define _CMSIS_NVIC_H_
+
+#include <stdint.h>
+#include <nrf.h>
+
+#define NVIC_NUM_VECTORS      (16 + 20)   /* CORE + MCU Peripherals */
```

Comment:
where this number (20) come from?

Reply:
yeap, miss that there are holes in irqs numbering (20 interrupts) but the largest one is 29